wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -1177,6 +1204,33 @@ public synchronized double getRate(IMUAxis axis) {
     return 0.0;
   }
 
+  /**
+   * Returns the Yaw axis angular rate in degrees per second (CCW positive).
+   *
+   * @return Yaw axis angular rate in degrees per second (CCW positive).
+   */
+  public synchronized double getRate() {
+    switch (m_yaw_axis) {
+      case kX:
+        if (m_simGyroRateX != null) {
+          return m_simGyroRateX.get();
+        }
+        return m_gyro_rate_x;
+      case kY:
+        if (m_simGyroRateY != null) {
+          return m_simGyroRateY.get();
+        }
+        return m_gyro_rate_y;
+      case kZ:
+        if (m_simGyroRateZ != null) {
+          return m_simGyroRateZ.get();
+        }
+        return m_gyro_rate_z;
+      default:
+    }
+    return 0.0;
```

Comment:
```suggestion
    return getRate(m_yaw_axis);
```

Reply:
I chose to not go with this as it has an extra switch case that can be avoided.